.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
- [ ] This comment about the README.md file, kindly consider using the README template provided by microverse